### PR TITLE
Added homepage

### PR DESCRIPTION
**Pull request by anamkaa**
https://user-images.githubusercontent.com/75376715/154726853-91f3d527-d873-4bee-9ba9-c22009d64efd.mp4

### CODE REVIEW

File: stylesheets/index.css [hunk 1]
```diff
@@ -1,38 +1,327 @@
...
+  box-shadow: var(--gh-footer-shadow);
+  gap: 1rem;
+  padding-top: 6rem;
+}
+
+.tab-section {
+  border-bottom: 2px solid var(--gh-bg2);
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (dev-enforced) [inline stylesheets/index.css:86]: These lines of code can be used frequently so you can make a separate utility class for it and then pass it into your HTML file.

**Review by dev-enforced — APPROVED** [review]
The code looks good to me!!Well structured code and use of variables in CSS classes. The home page also looks really good.

File: stylesheets/index.css (hunk 1 above)
Comment (anamkaa) [reply stylesheets/index.css:86]: Will implement it

File: index.html [hunk 2]
```diff
@@ -5,43 +5,297 @@
...
+       <div class="gh-filter-wrap-row">
+        <span class="h6">Rs. 1399</span> &nbsp;
+        <del class="h6">Rs. 1539</del> &nbsp;
       </div>
-      <div class="nav-icon-chip">
-        <i class="fas fa-cart-arrow-down"></i>
-        <span class="nav-icon-text">Cart</span>
+    </div>
+  </div></a>
+
```
Comment (Tanjum786) [inline index.html:172]: avoid blank spaces

**Review by Tanjum786 — APPROVED** [review]
overall code looks clean and clear. try to avoid spaces .and use beauty file or prettier. your home page looks nice great work.

File: index.html (hunk 2 above)
Comment (anamkaa) [reply index.html:172]: Will rectify it

**Review by Gautam2305 — COMMENTED** [review]
Code looks clean and good. Variables names are nicely assigned. Code looks clean. Good to merge.

**Pull request merged**